### DISCUSSION TITLE
mui slider not working inside in SwiperSlide

### DISCUSSION THREAD

**Discussion by MasterProvider** [nolimits4web/swiper#5910] [Q&A] [answered]
### Check that this is really a bug

- [X] I confirm

### Reproduction link

https://codesandbox.io/s/swiper-pagination-dynamic-react-forked-jt5od6?file=/src/App.jsx

### Bug description

MUI slider not working inside in SwiperSlide, onChange event still not work and no value anyone can tell me how to achieve this.

Thanks in advance

--updated--
I updated the swiper npm to the latest version and also I used preventDefault(), and stopPropagation(); why is this still not working anyone suggestion to achieve this behavior 

### Expected Behavior

I want to  MUI slider to work inside  in SwiperSlide

### Swiper version

8.3.2

### Platform/Target and Browser Versions

Chrome

### Validations

- [X] Follow our [Code of Conduct](https://github.com/nolimits4web/swiper/blob/master/CODE_OF_CONDUCT.md)
- [X] Read the [docs](https://swiperjs.com/swiper-api).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.
- [X] Make sure this is a Swiper issue and not a framework-specific issue

### Would you like to open a PR for this bug?

- [ ] I'm willing to open a PR

**Comment by muhaymin-bin-mehmood** [comment]
Did you find any solution ?

I am also stuck here

**Comment by MasterProvider** [comment, marked as answer]
Yes, by using
```
swiperRef.current.swiper.allowTouchMove=false
swiperRef.current.swiper.allowTouchMove=true
```

**Comment by maliuta-oleksandr** [comment]
there are no other options?

**Comment by lroyero333** [comment]
I also ran into this issue. Disabling the touch move for the swiper component did not work for me, but when testing with Chrome dev tools different devices, on devices with touch the slider works, but when I go back to devices without it it no longer works again.
Any help?
